beaker/client/api_providers_test.py

```diff
@@ -0,0 +1,49 @@
+from beaker.client.api_providers import (
+    Network,
+    Sandbox,
+    AlgoNode,
+    AlgoExplorer,
+    PureStake,
+)
+
+
+def test_sandbox():
+    sb = Sandbox(Network.SandNet)
+    sb.algod().suggested_params()
+    # sandbox indexer off
+    # sb.indexer().health()
```

Comment:
I assume this should be uncommented for release?

Reply:
im leaving it off for now, its sanity check and if you have indexer off on sandbox (i do) itll fail